projects/arrow/arrow/docker.py

```diff
@@ -309,19 +309,21 @@
 
 # RUST
 for arch in ['amd64']:
-    for rust_version in ['1.35']:
+    for rust_version in ['nightly-2019-11-14']:
```

Comment:
Wouldn't it be better to use `1.39` for this? I'm assuming that we already have good coverage of nightly on both Windows and Debian in the GH actions builds.

Reply:
Sure, if we intend to support that. I'll update the PR test and merge it.

Reply:
We need `1.39` or above to be able to use `async`, which we need to use for Flight but also we cannot build on stable Rust yet so require nightly releases and the current nightlies are `1.41` (with beta being `1.40`).

Reply:
I think I follow now, I never thought of why the build works on `1.35` when we require nightly. Looking at the logs from a recent build, we do use nightly. The label of 1.35 is what confused me :)

Reply:
Yep, that was the base image but cargo was using the toolchain defined in "arrow/rust/rust-toolchain"